xml/System.Threading.Channels/ChannelReader`1.xml

```diff
@@ -139,7 +139,7 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0;netcore-3.0;netcore-3.1;net-5.0;net-6.0" />
       </Parameters>
       <Docs>
-        <param name="cancellationToken">The cancellation token to use to cancel the enumeration.</param>
+        <param name="cancellationToken">The cancellation token to use to cancel the enumeration. Cancellation occurs after reading is finished.</param>
```

Comment:
I am not sure if "Cancellation occurs after reading is finished." is accurate description. Perhaps we should write something like this:

```suggestion
        <param name="cancellationToken">The cancellation token to use to cancel the enumeration. If there's already data available for reading immediately, cancellation occurs after reading is finished.</param>
```

@stephentoub do you have any opinion on that?

Reply:
Hi @adamsitnik your changes look reasonable to me.

Reply:
> cancellation occurs after reading is finished.

This part isn't quite right.  Cancellation might never occur, and I also don't want the docs to imply that it might read the value and then discard the value due to being canceled.

I think the sentiment this is trying to express is more like:
"If data is immediately available for reading, an implementation may prefer to yield that data immediately, even if cancellation has been requested."
Or something like that.

Reply:
Done!